com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/MatrixDeterminantNode.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class MatrixDeterminantNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "MatrixDeterminant",
+            "Out = determinant(In);",
+            new ParameterDescriptor("In", TYPE.Matrix, Usage.In, new float[] { 1f, 0f, 0f, 1f}),
+            new ParameterDescriptor("Out", TYPE.Float, Usage.Out)
+        );
+
+        public static Dictionary<string, string> UIStrings => new()
+        {
+            { "Name.Synonyms", "MatrixDeterminant" },
```

Comment:
Let's change this synonym to just "determinant"

Reply:
changed synonym to determinant